cdist/conf/type/__iptables_apply/files/init-script

```diff
@@ -24,14 +24,14 @@ case $1 in
         iptables-save > "$status"
 
         # Apply our ruleset
-        cd "$basedir"
-        count="$(ls -1 | wc -l)"
+        cd "$basedir" || exit
+        count="$(find . ! -name . -prune | wc -l)"
 
         # Only do something if there are rules
         if [ "$count" -ge 1 ]; then
             for rule in *; do
                 echo "Applying iptables rule $rule ..."
-                iptables $(cat "$rule")
+                iptables "$(cat "$rule")"
```

Comment:
This is another case where splitting is needed. The original line was correct/intentional.

Reply:
Ah, yes.